docs/source/get_involved/governance.rst

```diff
@@ -225,6 +248,16 @@ They are the point of contact for reporting potential CoC incidents.
 
 In addition, they are responsible for maintaining and improving the CoC.
 
+Expectations
+^^^^^^^^^^^^
+
+Without restriction of eligibility:
+* it is generally expected that CoC committee members
+  maintain a high degree of independence of day-to-day matters in the ``sktime`` community,
+  as well as high personal and professional standards of fairness.
+* it is highly advisable that CoC committee members are specifically selected for
+  being unaffected by any potential power dynamics within the ``sktime`` community.
+
```

Comment:
this is just some bla bla in my opinion. Its not binding or anything and does not provide a new information. I would remove it therefore.

Reply:
I think it's more than blabla.
Because it highlights what behaviours are expected of CoCC members, which in turn tells core developers what to look for in a vote.

But I'm also not to oattached to this part as it does not define process.

Reply:
replaced by the "should not be members of CC" which is more concrete